docs/browse/models.js

```diff
@@ -1741,13 +1774,23 @@ class Grant {
     if (this._impliedVisible != value) {
       this._impliedVisible = value;
       if (!value) {
-        this.filer.impliedVisible--;
-        this.grantee.impliedVisible--;
+        if (this.filer && typeof this.filer.impliedVisible === 'number') {
```

Comment:
ok, I think I get it. See, this was originally all booleans, and for Charity it is a boolean. When I realized I had to use reference counting, because if A->B and C->B, then B is visible unless explicitly hidden. So the typeof === 'number' probably isn't needed anymore, it just kept me from being confused. You're coming into this fresh, so you're like "CODE STINK!" So if everything is working for you, its probably good to go.